docs/howto/debugging.md

```diff
@@ -81,6 +82,13 @@ See also [in the "Troubleshooting" section below](#debug-remotely-never-loads).
 [chrome-devtools-device]: https://facebook.github.io/react-native/docs/debugging.html#debugging-on-a-device-with-chrome-developer-tools
 
 
+<div id="react-devtools">
+
+## React Devtools
+
+The standalone version of React DevTools, which is an electron app, can be used to inspect the react-native components of the app. Since the version of react-native we use is `0.59`, the latest version (v4) of react-devtools is not supported, and v3 needs to be installed.  This can be done using the command `npm i -g react-devtools@^3` See [this blog post](https://reactjs.org/blog/2019/08/15/new-react-devtools.html#which-versions-of-react-are-supported) for complete compability information. Refer to [it's npm page](https://www.npmjs.com/package/react-devtools) for instructions on how to use with React Native.
```

Comment:
Thanks for adding this section! We should have a section on this tool -- I was a bit surprised to see that we don't have one already. :slightly_smiling_face: 

As a matter of information architecture: the focus of this section should be documenting the fact that there *is* this tool, and what it's for.

That part doesn't need to be very long -- the first and last sentences you have here are basically enough. But two things to make it clearer:
* Should add a link to the section of the RN debugging doc: https://facebook.github.io/react-native/docs/debugging.html#react-developer-tools that discusses this tool.
* Those sentences that describe the tool as a whole should be their own paragraph, right at the start of this new section.

  Then the rest of what's here, about compatibility with different versions, should be a separate paragraph after that one. That's a more specific detail, so it should be separated to reflect that.